client/src/pages/Plan/Plan.js

```diff
@@ -16,6 +16,28 @@ const Plan = () => {
     const { student, realCourses, programme } = usePlan();
     const [searchTerm, setSearchTerm] = useState("");
     const [note, setNote] = useState("");
+    const start = 2020;
+    const end = 2021;
+    const years = [];
+    for (var i = start; i <= end; i++) {
+        years.push(i);
+    }
+    console.log(years);
```

Comment:
surely u can make this `const years = [2020, 2021]` lol

Reply:
A plan has a start and end year which this should represent

Reply:
Okay that's fine, can we get rid of console.log

Reply:
dooooooooone